### PR TITLE
OpenStackAttachmentPluginTest according to the new template definitions

### PR DESCRIPTION
**Pull request by ordansantos**

### CODE REVIEW

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java [hunk 1]
```diff
@@ -122,44 +129,53 @@ public void deleteInstanceThrowsRequestExceptionTest()
 
         this.openStackAttachmentPlugin.deleteInstance(instanceId, this.localToken);
     }
-
+    
+    // FIXME: is this test useful?
+    //test case: Check if a attachment returned by getInstance is not null when a valid instance is returned by getInstanceFromJson.
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java]: I think it is better to improve the test by assert the contents of attachmentInstance. I suggest you to delete the text and add a TODO mark indicating that.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java [hunk 2]
```diff
@@ -110,9 +116,10 @@ public void deleteInstanceTest() throws FogbowManagerException, ClientProtocolEx
         this.openStackAttachmentPlugin.deleteInstance(instanceId, this.localToken);
         
     }
-
+    
+    // FIXME: this method does nothing
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java]: I guess the idea was to actually forward the exception thrown in  this.client.doDeleteRequest(endpoint, localToken); I suggest you to delete the method and to add a TODO mark explaining that

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java [hunk 3]
```diff
@@ -61,30 +61,36 @@ public void setUp() {
...
+        
+        //verify
         Assert.assertEquals(FAKE_INSTANCE_ID, instanceId);
     }
-
+    
+    //test case: Check if requestInstance throws UnexpectedException when the http requisition throws UnexpectedException.
     @Test(expected = UnexpectedException.class)
-    public void requestInstanceThrowUnexpectedExceptionTest()
+    public void testRequestInstanceThrowUnexpectedException()
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java]: @ordansantos I suggest that you talk with the original author to understand the actual test case. After that, we can talk about it

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java [hunk 4]
```diff
@@ -122,44 +129,53 @@ public void deleteInstanceThrowsRequestExceptionTest()
...
     @Test
-    public void generateJsonToAttachTest() {
+    public void testGenerateJsonToAttach() {
+    	// setup
         VolumeOrder volumeOrder = Mockito.spy(new VolumeOrder());
         String volumeId = volumeOrder.getId();
-
         JSONObject json = this.openStackAttachmentPlugin.generateJsonToAttach(volumeId);
-
+
```
[the remark this answers is not in the document]
Comment (ordansantos) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java]: Makes sense.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java (hunk 1 above)
Comment (ordansantos) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java]: The test was properly rewritten.

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java (hunk 2 above)
Comment (ordansantos) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java]: The test was properly rewritten

File: src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java (hunk 3 above)
Comment (ordansantos) [reply src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java]: The test was properly rewritten

**Comment by ordansantos** [comment]
In addition to the requested changes, I have rewritten some methods.

**Pull request merged**